src/reducers/tasksReducer.js

```diff
@@ -52,6 +52,18 @@ export default function(state = initialState, action) {
         })
       });
 
+    case 'PATCH_TASK_PENDING':
+      return state;
+
+    case 'PATCH_TASK_FULFILLED':
+      const patchedTask = action.payload;
+      patchedTask.subtask_gids = state.taskMap[action.payload.gid];
+      return Object.assign({}, state, {
+        taskMap: Object.assign({}, state.taskMap, {
+          [action.payload.gid]: patchedTask
```

Comment:
@Moe-Shoman And now

`[action.payload.gid]: patchedTask`

can be

`[patchedTask.gid]: patchedTask`

Reply:
And I also made _this_ tweak in ee7f5673b877883903fda71f52274230f9e6026e